test/test_ripper_tags.rb

```diff
@@ -316,6 +316,55 @@ class C
     assert_equal '3: scope C.red',  inspect(tags[2])
   end
 
+  def test_extract_delegate
+    tags = extract(<<-EOC)
+      class C
+        delegate :foo,
+                 :bar, to: :thingy
+        delegate :x, to: :thingy, prefix: true
```

Comment:
That particular combination [is invalid](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/core_ext/module/delegation.rb#L162). Auto-prefixing works only if delegated to a method.